source/includes/administration/_organizations.md

```diff
@@ -470,6 +505,149 @@ Optional | &nbsp;
 
 Returns an HTTP status code 200, with an empty response body.
 
+<!-------------------- GET ORGANIZATION PASSWORD POLICY -------------------->
+
+### Get password policy
+`GET /organizations/organization_id/password_policy`
+
+Retrieve the password policy for the organization.
+
+```shell
+# Retrieve the organization's own or inherited password policy.
+curl "https://cloudmc_endpoint/api/v1/organizations/e8d95716-26a9-4054-833e-81cd3a5155cd/password_policy" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+    "data": [
+        {
+            "name": "min_password_length",
+            "value": 8,
+            "isMandatory": true
+        },
+        {
+            "name": "min_lowercase_letters",
+            "value": 1,
+            "isMandatory": true
+        },
+        {
+            "name": "min_uppercase_letters",
+            "value": 1,
+            "isMandatory": true
+        },
+        {
+            "name": "min_numbers",
+            "value": 1,
+            "isMandatory": true
+        },
+        {
+            "name": "min_special_characters",
+            "value": 1,
+            "isMandatory": true
+        }
+    ]
+}
+```
+Attributes | &nbsp;
+---- | -----------
+*Array[Object]*| A list of password policy constraint objects with the following fields.
+`name`<br/>*string* | The name of the constraint.
+`value`<br/>*int* | The minimum value for the constraint.
+`isMandatory`<br/>*boolean* | Flag to indicate if the constraint is mandatory or not.
+
+<!-------------------- CREATE/UPDATE ORGANIZATION PASSWORD POLICY -------------------->
+### Create/Update password policy
```

Comment:
Should we update the `Update security settings` section instead of creating a new section for password policy as it's the same endpoint?

Reply:
IMO, since `password policy` is totally optional during the create/update of `security settings`, keeping this `PUT` call lightweight would be idealistic. One use-case is when the user is interested in 
1. Only Creating/updating the security settings  --> document that `password policy` fields are optional
2. Only Creating/updating the password policy --> document that `security settings ` fields are optional

The API doc here would just become too conditional IMO, and segregating them (even though they belong to just one endpoint) might simplify it with the desired use case. 

But yes, if we want to document these two functionalities in one reference, I am cool with that as well!
Let me know!

Reply:
`if we want to document these two functionalities in one reference` - It would make sense to merge them in one because here we are documenting the APIs and not functionalities.

Similar to `autoCreationEnabled` and `verifiedDomains`, the password policy fields should go in the optional section.

Reply:
We should also add an `info` section something like - `If there is no default role provided in the request then Guest will be saved as default role.`